03/app.js

```diff
@@ -0,0 +1,15 @@
+const btns = document.querySelectorAll("button");
+
+const oneClickEvent = function () {
+  let counter = 0;
+  counter++;
+  this.innerText = "clicked";
+  console.log("clicked");
+  if ((counter = 1)) {
```

Comment:
Uwaga! Tu jest przypisanie `=` a nie porównanie `===` co zawsze będzie prawdą (jest sprawdzana wartość dla `counter`).
Akurat w tym przypadku działa, ale zazwyczaj jest to błąd. Dodatkowo niepotrzebna jest jedna para nawiasów ;)

Reply:
Ajj... Faktycznie. Jak to mawia klasyk: "fatalne przejęzyczenie" :D 
Poprawione. 
Nawiasy musiał mi dodać VSC kurcze.